spanner/leaderboard/step5/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.codelabs</groupId>
+  <artifactId>leaderboard</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
```

Comment:
Please add the parent pom to this project as well.

Reply:
Done.

Reply:
I don't see the parent in this pom.